data/mapreduce.py

```diff
@@ -0,0 +1,19 @@
+from pyspark import SparkContext
+
+sc = SparkContext("spark://spark-master:7077", "PopularItems")
+
+data = sc.textFile("/tmp/data/access.log", 2)     # each worker loads a piece of the data file
+
+pairs = data.map(lambda line: line.split(" "))   # tell each worker to split each line of it's partition
```

Comment:
I had issues with creating the input data in pycharm with tabs. I fixed it in my next commit by creating the input data in notepad